hello-world-module/values.yaml

```diff
@@ -11,7 +11,7 @@ image:
   pullPolicy: IfNotPresent
 imagePullSecrets: []
 # Action (copy/read)
-copy:
+assets:
   # Copy source
   source:
```

Comment:
There is a mismatch between declared values and the sent values. Manager sends an `assets `array, not a single structure.

Reply:
Fixed. Thanks